### PR TITLE
Deribit

### PR DESCRIPTION
**Pull request by kishoian**

### CODE REVIEW

File: content/attacks/posts/2022-11-01-Deribit.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-11-01
+categories: Custodian
```
Comment (eric-oconnor) [inline content/attacks/posts/2022-11-01-Deribit.md:3]: Add quotes around the Categories: "Type of Attack"

**Review by eric-oconnor — CHANGES_REQUESTED** [review]

**Review by eric-oconnor — COMMENTED** [review]
/qualitycheck

**Comment by jhirschkorn** [comment]
Double check dates and reconcile with bot comments

File: content/attacks/posts/2022-11-01-Deribit.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Losses
+
+The attackers managed to steal [~691 Bitcoin (BTC)](https://www.blockchain.com/ru/explorer/transactions/btc/b8423bb78f26be3d4c0a5dec70c46885c24307bf574f9c941ed37d73ae29f7a2) and [~6,947 Ether (ETH)](https://etherscan.io/tx/0xa1822e68a736bcdb57d05b2679260904813efdd17df62ede1d716dec9eeb4e8c) and [~$3,394,823​​ USDC](https://etherscan.io/tx/0x9ae755bfbb181cc991fc2d54ec6ab04f331042cea5d33e95476846446cf88815) from the hot wallet, worth approximately $28 million at the time of the attack.
+
+## Timeline
+
+- **November 1, 2022:** The Deribit hack begins. The attacker gains access to the exchange's hot wallet and withdraws 691 BTC and 9,080 ETH.
+- **November 1, 2022:** [Deribit halts withdrawals and announces the hack.](https://twitter.com/DeribitExchange/status/1587701883778523136)
+- **November 2, 2022:** Deribit says it will cover the losses from the hack using its own funds.
```
Comment (JediFaust) [inline content/attacks/posts/2022-11-01-Deribit.md:31]: Provide source please

**Review by eric-oconnor — APPROVED** [review]
quotes added

**Pull request merged**